Dockerfile

```diff
@@ -32,6 +32,5 @@ RUN bundle exec rake assets:precompile && \
 ARG COMMIT_SHA
 ENV COMMIT_SHA=$COMMIT_SHA
 
-CMD bundle exec rails db:migrate:ignore_concurrent_migration_exceptions && \
-    bundle exec rails data:migrate && \
+CMD bundle exec rails db:migrate:with_data_migrations && \
```

Comment:
I think we still need to tie this in with the ignore concurrent migrations thing or else when we deploy to prod we'll get and error thrown from each box that tries to run the migrations once they've started. I wonder about whether there's also a race condition where two boxes run the migration before it gets 'locked'. I wonder whether it would be safer to only have one instance (or a separate instance) or something that just runs the DB stuff. In Capistrano  we used to just assign the DB role to one box. They were the days 🤣

Reply:
Those indeed were the days! I've moved the concurrent stuff into this new script https://github.com/DFE-Digital/register-trainee-teachers/pull/1386/files#diff-58b8ed21bb86690bade60dc7dda66476dcdfcbcd02dc976937c291948f7e5cd6R13

Reply:
Sorry... Missed that. Perfect 👍